### PR TITLE
Fixes for JOSS manuscript

### PR DESCRIPTION
**Pull request by gdmiron**

### CODE REVIEW

File: paper/paper.bib [hunk 1]
```diff
@@ -30,7 +30,7 @@ @article{Miron2019
...
-   author = {G. D. Miron and Dmitrii A. Kulik and Tres Thoenen},
+   author = {G.bell D. Miron and Dmitrii A. Kulik and Tres Thoenen},
```
Comment (allanleal) [inline paper/paper.bib:33]:
```suggestion
   author = {G. D. Miron and Dmitrii A. Kulik and Tres Thoenen},
```

File: paper/paper.md [hunk 2]
```diff
@@ -30,40 +30,40 @@ bibliography: paper.bib
...
...a range of temperatures and pressures. The library is developed in C++ for performance, but it also has a Python API for broader and more convenient usage. It employs a variety of thermodynamic models and equations of state for solid, aqueous, surface, gaseous, and molten substances, and their reactions with input properties and parameters from various thermodynamic datasets that are collected and curated in the in [ThermoHub](https://thermohub.org/thermohub) database. The library can be used as a standalone code for searching and tabulating thermodynamic properties or linked to other modeling codes that require standard thermodynamic data as input. It offers the flexibility to use different thermodynamic datasets, including custom datasets and datasets retrieved from the online ThermoHub database, and to choose the most suitable models for different classes of substances necessary in various modeling applications. It can serve as a common source of thermodynamic models for standard properties of substances and reactions that can be easily integrated and combined, significantly improving the modeling capabilities for diverse (geo)chemical systems and over wide ranges of conditions.
```
Comment (allanleal) [inline paper/paper.md:33]: Should the word *standard* be used before *thermodynamic models and equations of state*?

**Review by dimitrikulik — APPROVED** [review]
Reasonable changes and corrections, can be merged now.

File: paper/paper.md (hunk 2 above)
Comment (allanleal) [reply paper/paper.md:33]: Let's leave this in general - "thermodynamic models". No change needed here. Thanks for the explanation, Dmitrii.

**Review by allanleal — APPROVED** [review]
All looks good to me!

**Comment by gdmiron** [comment]
Thanks for the comments @allanleal and @dimitrikulik. I will have to convince the editor to allow a normal length manuscript and not a one page "elevator pitch" as it was suggested.

**Pull request merged**